webapp/views/wpt-app.js

```diff
@@ -68,13 +68,39 @@ class WPTApp extends PathInfo(WPTFlags(TestRunsUIBase)) {
           padding: 0px;
           height: 28px;
         }
+
+        /* TODO(DanielRyanSmith): Remove these when interop 2025 proposals are closed. */
+        .interop-2025-banner {
+          height: 40px;
+          background-color: #DEF;
+          text-align: center;
+          padding-top: 16px;
+          border: 2px solid #1D79F2;
+          border-radius: 8px;
+        }
+        .interop-2025-banner p {
+          margin: 0;
+          font-size: 18px;
+        }
+        .interop-2025-banner a {
+          color: #0d5de6;
+          text-decoration: none;
+        }
       </style>
 
       <app-location route="{{route}}" url-space-regex="^/(results)/"></app-location>
       <app-route route="{{route}}" pattern="/:page" data="{{routeData}}" tail="{{subroute}}"></app-route>
 
       <wpt-header path="[[encodedPath]]" query="[[query]]" user="[[user]]" is-triage-mode="[[isTriageMode]]"></wpt-header>
 
+      <a href="https://example.com" target="_blank">
```

Comment:
You may want to add a similar comment here to remove this anchor tag when interop 2025 proposals are closed.